katalog/aws-node-termination-handler/kustomization.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: kube-system
+
+resources:
+  - daemonset.yml
+  - rbac.yml
+  - podmonitor.yml
+
+images:
+  - name: public.ecr.aws/aws-ec2/aws-node-termination-handler
+    newName: public.ecr.aws/aws-ec2/aws-node-termination-handler
```

Comment:
We have to push this image to our registry, then use the new image name here

Reply:
yep true, but idk how to do it, in the meanwhile, I set it up in the right way to make flawless the migration once the image is on our registry

Reply:
Feel free to read: https://github.com/sighupio/container-image-sync